docs/source/en/_toctree.yml

```
@@ -6,21 +6,49 @@
   - local: installation
     title: Installation
   title: Get started
+
 - sections:
-  - local: tutorials/placeholder
-    title: Placeholder
+  - local: tutorials/overview
+    title: Overview
+  - local: tutorials/zero_shot
+    title: Zero-shot Text Classification
   title: Tutorials
+
 - sections:
-  - local: how_to/placeholder
-    title: Placeholder
+  - local: how_to/overview
+    title: Overview
+  - local: how_to/callbacks
+    title: Callbacks
+  - local: how_to/classification_heads
+    title: Classification Heads
+  - local: how_to/multilabel
+    title: Multilabel Text Classification
+  - local: how_to/zero_shot
+    title: Zero-shot Text Classification
+  - local: how_to/hyperparameter_optimization
+    title: Hyperparameter Optimization
+  - local: how_to/knowledge_distillation
+    title: Knowledge Distillation
+  - local: how_to/batch_sizes
+    title: Batch Sizes
```

small suggestion to make it clearer
```suggestion
    title: Batch Sizes for Inference
```

---

Resolved in 2ef61bb1ff55781d966bde032715260224006878